Guessing the random number project

vinay-s36 opened the pull request:
#### Developer Checklist
- [✅] Followed guidelines mentioned in the readme file.
- [✅] Followed directory structure. (e.g. ProjectName/{USERNAME}/...yourfiles)
- [✅] Starred ⭐ the Repo (Optional)

#### Summary
###### Guess It - The Random Number Guesser project in JavaScript is an interactive web application that challenges users to guess a randomly generated number within a specified range. It provides an engaging way to learn and practice JavaScript programming while creating a fun user experience.

#### Screenshot
![image](https://github.com/thinkswell/javascript-mini-projects/assets/124019116/bc76a0db-5a8e-4e7c-ba68-1d7a349f2039)

![image](https://github.com/thinkswell/javascript-mini-projects/assets/124019116/057b5532-dd81-4b24-9b60-790a62b1ee7f)

![image](https://github.com/thinkswell/javascript-mini-projects/assets/124019116/661e2419-e582-40f2-92de-81b1adfe86b8)

#### Live Project Link
###### https://vinay-s36.github.io/Guess-It/

PBJI reviewed (changes requested):
Hi @vinay-s36 , your repository requires some changes before it can be merged:

<hr>




❌ RENAMING: You should considering adding your project in the pre-existing **NumberGuessingGame** folder and not **Guess It** folder.

<hr>



❌ DIRECTORY: Your project files, including the README.md should be inside another folder called **vinay-s36**, so ideally your pull request changes would look something like this:
```
javascript-mini-projects
|           
-------- NumberGuessingGame
           |
            ----- vinay-s36
                   |
                    -------- **Your project files and folders**
                   |
                    --------- README.md (optional yet preferred)
```

<hr>



😅 ADDING (Optional): And consider adding/updating README.md with 
1) Summary about your project
2) How to setup and use 
3) Add the screenshots in it.

<hr>




❓ REFERENCES:

1) Refer to [How to raise the PR](https://github.com/thinkswell/javascript-mini-projects#steps-to-raise-and-get-your-pr-merged-successfully) for more information

<hr>


Once you make these necessary changes, please revert back with a comment mentioning @PBJI so that I get notified.




Thank you. 😄

vinay-s36 commented:
@PBJI i have made all the necessary changes i was told

added files in the same structure i was asked and added READme.md with instructions of how to run the project with screenshots of the peoject

Let me know if there are any changes required
